test/unit/all.spec.js

```diff
@@ -35,6 +35,10 @@ for (const test of window.tests) {
       // await svg2pdf(svgElement, pdf, svg2pdfOptions)
 
       comparePdf(pdf.output(), `/test/specs/${name}/reference.pdf`, debug)
+
+      if (document.querySelector('svg')) {
+        expect.fail('svg element left')
```

Comment:
"svg element left" sounds like "svg element just left the room" :D
Maybe something like: "svg measuring element must not remain in document"